src/common/_modules/application_gateway/main.tf

```diff
@@ -503,6 +512,20 @@ module "app_gw" {
       rewrite_rule_set_name = "rewrite-rule-set-fims-op-app"
       priority              = 120
     }
+
+    vehicles-ipatente-io-pagopa-it = {
+      listener              = "vehicles-ipatente-io-pagopa-it"
+      backend               = "vehicles-ipatente-io-app"
+      rewrite_rule_set_name = "rewrite-rule-set-vehicles-ipatente-io-app"
+      priority              = 130
+    }
+
+    licences-ipatente-io-pagopa-it = {
+      listener              = "licences-ipatente-io-pagopa-it"
+      backend               = "licences-ipatente-io-app"
+      rewrite_rule_set_name = "rewrite-rule-set-licences-ipatente-io-app"
```

Comment:
Can we use `local.io_backend_ip_headers_rule`?

Reply:
8f8ecd8c02bb0fdf9613ae58dd7bffe509f8e41b